src/common/containers/landing.js

```diff
@@ -2,25 +2,78 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 
 import { Anchor } from '../components/vanilla/button';
+import { HeadingOne } from '../components/vanilla/heading';
 
-import styles from './container.css';
+import containerStyles from './container.css';
+import styles from './landing.css';
+
+import 'isomorphic-fetch';
 
 class Landing extends Component {
   render() {
     return (
-      <div className={ styles.container }>
-        <h2>Auto-build and publish software for any Linux system.</h2>
-        <p>Just a placeholder landing page...</p>
-        <Anchor href="/auth/authenticate">Set Up in Minutes</Anchor>
+      <div>
+        <div className={ containerStyles.lightStrip }>
+          <div>
+            <div className={ containerStyles.wrapper }>
+              <HeadingOne>
+              Auto-build and publish software for any Linux system or device.
+              </HeadingOne>
+
+              <div className={ styles.bannerList }>
+                <div className={ styles.bannerItem }>
+                  <img src="http://placehold.it/200" />
+                  <p>Push to GitHub.</p>
+                </div>
+                <div className={ styles.bannerItem }>
+                  <img src="http://placehold.it/200" />
+                  <p>Built automatically.</p>
+                </div>
+                <div className={ styles.bannerItem }>
+                  <img src="http://placehold.it/200" />
+                  <p>Published for your users.</p>
+                </div>
+              </div>
+
+              <div className={ styles.bannerButton }>
+                <Anchor  href="/auth/authenticate">Set Up in Minutes</Anchor>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className={ styles.container }>
+
+          {/* TODO brand logos */}
+
+          {/* TODO workflow */}
+
+          <div>
+            {/* TODO 2 columns */}
+            <ul>
+              <li>Scale to millions of installs</li>
+              <li>Available on all clouds and Linux OSes</li>
+              <li>No need for build infrastructure</li>
+              <li>Automatic updates for everyone</li>
+              <li>Roll back versions effortlessly</li>
+              <li>FREE for open source projects</li>
+            </ul>
+          </div>
+
+          <div>
+            <p>Snaps are secure, sandboxed, containerised applications, packaged with their dependencies for predictable behavior. With the Snap Store, people can safely install apps from any vendor on mission-critical devices and PCs.</p>
+            <Anchor href="http://snapcraft.io">More about snaps</Anchor>
+            {/* TODO testimonials */}
+          </div>
+        </div>
       </div>
     );
   }
+
 }
 
 Landing.propTypes = {
   children: PropTypes.node,
-  auth: PropTypes.object,
-  router: PropTypes.object
+  auth: PropTypes.object
```

Comment:
`auth` is not used at the moment (it used to be when redirecting authorised users happened here), but I keep it around as I assume at some point we may want to show different text on landing page for authorised users (it's mentioned in a doc for future versions).